src/components/SecretsTable/Secrets.scss

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright 2019 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+@import '../../scss/vars';
+
+.notificationComponent {
+  position: absolute;
+  min-width: 30rem;
+  margin-bottom: 0.5rem;
+  z-index: 1000;
+  right: 0;
+  top: 0;
+}
+
+.secrets {
+  svg {
+    height: 24px;
```

Comment:
svg size is determined by the component that's used, e.g. `@carbon/icons-react/lib/chevron--right/24`
we should avoid setting / overriding the size in the styles as many of the svgs have been optimised for specific sizes and changing this can result in strange rendering

Reply:
the reason behind overriding the size is because i tried importing `@carbon/icons-react/lib/close--outline/24` but got an `Unable to resolve path to module` error. Any other way I can achieve this @AlanGreene ?

Reply:
ok let's use the css to set the size for now